xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/internal/html/DefaultHTMLCleanerTest.java

```diff
@@ -457,6 +456,7 @@ void cleanHTMLTagWithNamespace()
      * check, see <a href="https://github.com/cure53/DOMPurify/commit/e32ca248c0e9450fb182e52e978631cbd78f1123">commit
      * e32ca248c0 in DOMPurify</a>.
      */
+    // FIXME: Test not passing anymore, apparently the template is surviving now.
```

Comment:
@michitux I need your opinion on this one: apparently test is not passing anymore in HTML5CleanerTest: the template tag is still there in the output.

Reply:
Maybe the tag definition in HTMLCleaner has been changed to be less strict? Is this still relevant (I see that you've closed the PR)?

Reply:
Yes I closed that PR because I moved the work to another branch to allow building this on CI, see: https://github.com/xwiki/xwiki-commons/pull/799#discussion_r1507810696